src/components/dev-hub/podcast-player/player-contols.js

```diff
@@ -0,0 +1,98 @@
+import React, { memo } from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import { css } from '@emotion/react';
+
+import Button from '~components/dev-hub/button';
+import FastForwardIcon from '~components/dev-hub/icons/audio-player/fast-forward-icon';
+import PauseIcon from '~components/dev-hub/icons/audio-player/pause-icon';
+import PlayIcon from '~components/dev-hub/icons/audio-player/play-icon';
+import RewindIcon from '~components/dev-hub/icons/audio-player/rewind-icon';
+
+import { size } from '~components/dev-hub/theme';
+
+const buttonStyles = css`
+    margin-right: 12px;
```

Comment:
For the fast forward button we might not always want the margin. Here it is on mobile (not quite centered):

<img width="410" alt="Screen Shot 2021-04-28 at 3 45 06 PM" src="https://user-images.githubusercontent.com/9064401/116463570-d3cd1580-a838-11eb-8c7a-0d1d16bd1809.png">